main_server.py

```
@@ -2041,6 +2829,1187 @@ def search_files_recursive(directory, target_ext, result_list):
+            "success": False,
+            "error": "Steamworks API错误",
+            "message": "请确保Steam客户端已运行且已登录"
+        }, status_code=503)
+    except Exception as e:
+        logger.error(f"发布到创意工坊失败: {e}")
+        return JSONResponse(content={"success": False, "error": str(e)}, status_code=500)
+
+@app.post('/api/steam/workshop/test-file-handling')
+async def test_file_handling(request: Request):
+    """
+    测试文件处理功能的端点，用于验证路径规范化、文件夹检查等功能是否正常工作
```

Ruff (0.14.5)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)